### PR TITLE
fix(forms): safari/phantom don't have setImmediate

### PR DESCRIPTION
**Pull request by jefflembeck**

### CODE REVIEW

File: src/pivotal-ui/components/forms/link-updater.js [hunk 1]
```diff
@@ -20,14 +20,14 @@ LinkUpdater.prototype.updateValue = function(inputValue){
...
+    // setTimeout 0 forces input-error to be triggered on the next tick, so it
     // will follow any input
-    setImmediate(function(){
+    setTimeout(function(){
       this.input.trigger({
         type: "input-error",
         message: err.message
       });
-    }.bind(this));
+    }.bind(this), 0);
```
Comment (aredridel) [inline src/pivotal-ui/components/forms/link-updater.js]: since 0 is 4ms and definitely the next render cycle, is requestanimationframe a better option?

Comment (jefflembeck) [inline src/pivotal-ui/components/forms/link-updater.js]: I thought `setImmediate(fn)` was the exact equivalent to `setTimeout(fn, 0)`?

Comment (jefflembeck) [inline src/pivotal-ui/components/forms/link-updater.js]: RAF _might_ be better here though. Just trying to reduce complexity whichever way possible.

Comment (jefflembeck) [inline src/pivotal-ui/components/forms/link-updater.js]: Ok, flipped to RAF

Comment (aredridel) [inline src/pivotal-ui/components/forms/link-updater.js]: setImmediate is equivalent _on node_.

In the browser, it bypasses the 4ms delay of settimeout 0.

**Comment by aredridel** [comment]
LGTM!

**Pull request merged**